events/events.go

```diff
@@ -454,3 +453,10 @@ type TeamSideSwitch struct {
 // GameHalfEnded is usually dispatched in the end of a round, just before RoundEndOfficial.
 type GameHalfEnded struct {
 }
+
+// MatchStartChanged signals that the value of data table DT_GameRulesProxy.m_bHasMatchStarted has changed
+// This can be useful for some demos where the MatchStart event is not sent.
+type MatchStartChanged struct {
```

Comment:
Should be named `MatchStartedChanged` or just `MatchStartedChange` to match the variable/function/attribute names. I'd prefer the former in this case (in combination with my other comment about moving the dispatching down, as it will then be the correct tense).